Bio/Affy/CelFile.py

```diff
@@ -1,10 +1,10 @@
-# Copyright 2004 by Harry Zuzan.  All rights reserved.
+# Copyright 2004 by Harry Zuzan and Adam Kurkiewicz. All rights reserved.
```

Comment:
Your work is from 2016, so I'd use a second line following the style elsewhere in Biopython:

``` python
# Copyright 2004 by Harry Zuzan.  All rights reserved.
# Copyright 2016 by Adam Kurkiewicz. All rights reserved.
```